### PR TITLE
use new Ragger and workflows compatible with plugin testing

### PR DESCRIPTION
**Pull request by fbeutin-ledger**
Using https://github.com/LedgerHQ/ledger-app-workflows/pull/44

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -1,53 +1,63 @@
...
+On Linux, you can use [Ledger's VS Code extension](#with-vscode) to run the tests.
+
+Alternatively, you can run the test by command lines in your terminal.
 
-Install the tests requirements :
+First, run this command once to install the tests requirements:
 
 ```shell
-pip install -r tests/requirements.txt 
+pip install -r tests/requirements.txt --extra-index-url
```
Comment (tdejoigny-ledger) [inline tests/README.md]: Probably better to remove `--extra-index-url` and to add these lines in requirements.txt
```
--index-url https://test.pypi.org/simple/
ledger_app_clients.ethereum
```

File: tests/requirements.txt [hunk 2]
```diff
@@ -2,5 +2,4 @@ pytest
 ragger[speculos,ledgerwallet]>=1.11.4
 ecdsa
 simple-rlp
-# to change when ethereum client will be pushed on master
-tests/client/
\ No newline at end of file
+ledger_app_clients.ethereum
```
Comment (tdejoigny-ledger) [inline tests/requirements.txt]:
```
--index-url https://test.pypi.org/simple/
ledger_app_clients.ethereum
```

Comment (tdejoigny-ledger) [inline tests/requirements.txt]: It seems that there are some conflicts between python packages when running tests with the CI.

**Comment by tdejoigny-ledger** [comment]
This file needs to be removed : `tests/keychain.py`

**Review by tdejoigny-ledger — COMMENTED** [review]
This file needs to be removed : `tests/keychain.py`

File: tests/README.md (hunk 1 above)
Comment (fbeutin-ledger) [reply tests/README.md]: I thought we decided to keep the command line to be explicit ? And handle it in readme (for manual usage) and vscode plugin
cc @apaillier-ledger

File: tests/requirements.txt (hunk 2 above)
Comment (fbeutin-ledger) [reply tests/requirements.txt]: solved

**Pull request merged**